Source/bmalloc/libpas/src/libpas/pas_lock.h

```diff
@@ -194,7 +199,12 @@ static inline bool pas_lock_try_lock(pas_lock* lock)
     bool result;
     if (PAS_LOCK_VERBOSE)
         pas_log("Thread %p Trylocking lock %p\n", pthread_self(), lock);
+#if PAS_USE_ULOCK_SPI && (!defined(OS_UNFAIR_LOCK_INLINE) || !OS_UNFAIR_LOCK_INLINE)
+    result = false;
+    PAS_ASSERT_NOT_REACHED();
```

Comment:
Your commit message says this is a release assert. `PAS_ENABLE_ASSERT` seems a bit more tricksy than that, it seems to depend on the platform. I assume you're aware, just checking.